suite/tests/CMakeLists.txt

```diff
@@ -2693,9 +2693,11 @@ if (X86 OR AARCH64) # FIXME i#1551: port asm to ARM
     tobuild_ci(client.flush client-interface/flush.c "" "" "")
```

Comment:
Re the PR description:

Please add a proper description, even if it somewhat repeats the title: https://dynamorio.org/page_code_reviews.html.